app/components/Analyst/NavItem.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+interface Props {
+  href?: string;
+  icon?: any;
+  label: string;
+  currentPath: string;
+}
+
+interface StyledProps {
+  selected: any;
+}
+
+const StyledNavItem = styled.div<StyledProps>`
+  margin: 4px 0;
+  padding: 8px;
+  display: flex;
+  border-radius: 4px;
+  background-color: ${(p) => p.selected};
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.color.navigationLightGrey};
+  }
+`;
+
+const StyledLink = styled.a`
+  color: inherit;
+  text-decoration: none;
+  padding-left: 8px;
+`;
+
+const StyledIconContainer = styled.div`
+  width: 12px;
+`;
+
+const StyledLinkContainer = styled.div`
+  display: none;
+
+  ${(props) => props.theme.breakpoint.largeUp} {
+    display: block;
+  }
+`;
+
+const NavItem: React.FC<Props> = ({ currentPath, href, icon, label }) => {
+  const selectedColour = currentPath === href ? '#F1F2F3' : 'inherit';
+
+  return (
+    <StyledNavItem selected={selectedColour}>
+      <StyledIconContainer>
+        <FontAwesomeIcon icon={icon} fixedWidth aria-hidden="true" />
```

Comment:
When the link is hidden, if you click on an icon nothing happens. If you add another <Link> around here that should remedy the issue

Reply:
Fixed. Nice catch, really appreciate that :bow: